network/p2p/network_test.go

```diff
@@ -480,141 +560,58 @@ func TestAppRequestDuplicateRequestIDs(t *testing.T) {
 	require.ErrorIs(err, ErrRequestPending)
 }
 
-// Sample should always return up to [limit] peers, and less if fewer than
-// [limit] peers are available.
-func TestPeersSample(t *testing.T) {
```

Comment:
Sample is no longer exported out of the package so we're testing this through `AppRequestAny` now